frontend/e-buddy/src/app/(demo)/bots/setup/page.tsx

```
@@ -0,0 +1,159 @@
+              <h2 className="text-2xl font-semibold text-gray-800 mb-4">Step 1: Information</h2>
+              <form className="space-y-6">
+                <div>
+                  <label className="block text-gray-700 mb-2">Bot Name</label>
+                  <input 
+                    type="text" 
+                    placeholder="Enter bot name" 
+                    className="w-full border border-gray-300 rounded-lg px-4 py-3 shadow-sm focus:outline-none focus:ring-2 focus:ring-blue-500 transition"
+                  />
+                </div>
+                <div>
+                  <label className="block text-gray-700 mb-2">Model</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)